docker-compose.yml

```diff
@@ -1,5 +1,23 @@
 db:
   image: postgres:9.3
+broker:
+  image: rabbitmq
+celery:
+  build: .
+  volumes:
+    - .:/code
+  links:
+    - db
+    - broker
+  environment:
+    - PYTHONDONTWRITEBYTECODE=1
```

Comment:
NIT: I would prefer all the django env settings to be defined in .env instead since these are per case. Eg. its not a good practice to hard code SECRET_KEY (even for dev).